src/dapps/SecureTransaction/SecureTransaction.vue

```diff
@@ -0,0 +1,460 @@
+    return {
+      protectionLevel: 'low',
+      advancedExpend: false,
+      validAmount: false,
+      validNetwork: false,
+      validAddress: false,
+      chainIDEnglish: CoralConfig.chainIDEnglish || 'mainnet',
+      minimumAmount: 0.01,
+      amount: 0,
+      safeSendPriceEstimate: 0,
+      amountValid: true,
+      nonce: 0,
+      gasLimit: 21000,
+      data: '0x',
+      gasAmount: 0,
+      parsedBalance: 0,
+      address: '',
+      transactionFee: 0,
+      selectedCurrency: { symbol: 'ETH', name: 'Ethereum' },
+      raw: {},
+      signedTx: '',
+      resolvedAddress: ''
+    };
+  },
+  computed: {
+    ...mapGetters({
+      account: 'account',
+      gasPrice: 'gasPrice',
+      web3: 'web3',
+      wallet: 'wallet',
+      network: 'network',
+      ens: 'ens'
+    })
+  },
+  watch: {
+    address(newVal) {
+      this.address = newVal;
+      if (this.verifyAddr()) {
+        this.validAddress = false;
+      } else {
+        this.estimateGas();
+        this.validAddress = true;
+      }
+    },
+    amount() {
+      this.validAmount =
+        this.amount >= parseFloat(this.minimumAmount, 10) &&
+        this.amount <= parseFloat(this.parsedBalance, 10);
+    },
+    parsedBalance(newVal) {
+      this.parsedBalance = newVal;
+    },
+    gasAmount(newVal) {
+      this.gasAmount = newVal;
+      if (this.verifyAddr()) {
+        this.estimateGas();
+      }
+      this.$store.dispatch('setGasPrice', Number(newVal));
+    },
+    selectedCurrency(newVal) {
+      this.selectedCurrency = newVal;
+      this.estimateGas();
+    }
+  },
+  mounted() {
+    const address = this.$store.state.wallet.getAddressString();
+
+    this.web3.eth
+      .getBalance(address)
+      .then(res => {
+        this.balance = this.web3.utils.fromWei(res, 'ether');
+        this.$store.dispatch('setAccountBalance', this.balance);
+        this.parsedBalance = this.balance;
+        this.gasAmount = this.gasPrice;
+      })
+      .catch(err => {
+        // eslint-disable-next-line no-console
+        console.error(err);
+      });
+    this.web3.eth.net
+      .getId()
+      .then(res => {
+        this.networkID = res;
+        this.validNetwork = this.networkID === CoralConfig.chainID;
+      })
+      .catch(err => {
+        // eslint-disable-next-line no-console
+        console.error(err);
+      });
+  },
+  methods: {
+    debouncedAmount: utils._.debounce(function(e) {
+      this.amount = parseFloat(
+        new BigNumber(e.target.value).decimalPlaces(18).toFixed(),
+        10
+      );
+      this.getSafeSendFee();
+      if (this.verifyAddr()) {
+        this.estimateGas();
+      }
+    }, 501),
+    debounceInput: utils._.debounce(function(e) {
+      this.address = normalise(e.target.value);
+    }, 110),
+    copyToClipboard(ref) {
+      this.$refs[ref].select();
+      document.execCommand('copy');
+    },
+    async createTx() {
+      this.nonce = await this.$store.state.web3.eth.getTransactionCount(
```

Comment:
since web3 has been imported through the getters, you can just shorten this by using `this.web3.eth`